src/Components/FormWrapper/index.test.js

```diff
@@ -1,10 +1,30 @@
 import React from "react";
-import { mount } from "enzyme";
+import { render, fireEvent, wait } from "@testing-library/react";
 import FormWrapper from "./index";
 
+const mockSubmitFn = jest.fn();
+
+const Button = () => <button type="submit">Submit</button>;
+
 describe("FormWrapper component", () => {
-  test("should render properly", () => {
-    const wrapper = mount(<FormWrapper onSubmit={() => {}} />);
```

Comment:
Is there no longer a way to check if a component simply renders or not using react's testing library?

Reply:
> Is there no longer a way to check if a component simply renders or not using react's testing library?

React Testing Library has a similar method called `render`. However, only calling the `render` method is not enough to test the component, because it is only used to render the component into the DOM. We need to write test assertion to verify somethings like expect('some text') in the document or get button element to fire click event to expect some behavior happens.

Reply:
Ok, as long as all components include at least one test that does something with the DOM I suppose the "Should render" test will be accounted for.